File: EmgenexBusinessPortal/BusinessPanel/src/app/models/report-column-model.ts
export class ReportColumnModel {
    constructor(
        public Id: number = 0,
        public serviceId: number = 0,
        public ColumnName: string = null,
        public DisplayName: string = null,
        public ColumnType: number = 0,
        public InputType: number = 0,
        public IsMandatory: boolean = false,
        public DisplayInFilter: boolean = false,
        public ShowInGrid: boolean = false,
        public RolePrivilegeIds: number[] = [],
        public DepartmentPrivilegeIds: number[] = [],
        public UserPrivilegeIds: number[] = [],
        public ServiceName: string = null,
        public CreatedOn: DateTimeFormat = null,
        public UpdatedOn: DateTimeFormat = null,
        public CreatedByName: string = "",
        public UpdatedByName: string = "",
    ) { }
}

export class ReportColumnFilter {
    constructor(
        public KeyWords: string = null,
        public pageSize: number = 25,
        public currentPage: number = 1,
        public SortKey: string = null,
        public SortOrder: string = null,
    ) { }
}

File: EmgenexBusinessPortal/Areas/Business/Models/ReportColumnModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace EmgenexBusinessPortal.Areas.Business.Models
{
    public class ReportColumnModel 
    {
        // GET: Admin/ReportColumnModel
        public int Id { get; set; }
        public int BusinessId { get; set; }
        public int ServiceId { get; set; }
        public string ColumnName { get; set; }
        public int? ColumnType { get; set; }
        public bool? IsMandatory { get; set; }
        public bool? DisplayInFilter { get; set; }
        public DateTime CreatedOn { get; set; }
        public int CreatedBy { get; set; }
        public DateTime? UpdatedOn { get; set; }
        public int? UpdatedBy { get; set; }
        public int[] DepartmentPrivileges { get; set; }
        public int[] RolePrivileges { get; set; }
        public int[] UserPrivileges { get; set; }
        public bool? IsDeny { get; set; }
        public bool? ShowInGrid { get; set; }
        public string DisplayName { get; set; }
        public int? InputType { get; set; }
    }

    public class LookupServiceColumnsModel
    {
        public int Id { get; set; }
        public int ColumnId { get; set; }
        public string Text { get; set; }
        public int? Value { get; set; }
        public DateTime? CreatedOn { get; set; }
        public int CreatedBy { get; set; }
        public DateTime? UpdatedOn { get; set; }
        public int? UpdatedBy { get; set; }
    }
}